spec/menu_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'menu'
+
+describe 'Menu' do
+  it 'lists menu items' do
+    menu_hash = {'fish':[5.0, 0],
```

Comment:
Define this sort of thing either using `let` or as a separate method later in the test.